### PR TITLE
Fixed some more typos

### PR DESCRIPTION
**Pull request by agaida**
but a native speaker (or similar) review is urgently needed

### CODE REVIEW

**Comment by paulolieuthier** [comment]
GTM.

File: man/lxqt-policykit-agent.1 [hunk 1]
```diff
@@ -6,13 +6,13 @@ lxqt-policykit-agent \- Polkit agent for \fBLXQt\fR: the faster and lighter QT D
 [\fICurrently No command line arguments\fR]
 .br
 .SH DESCRIPTION
-This module handles policykit for \fBLXQt\fR. The polkit module, will perform 
+This module handles policykit for \fBLXQt\fR. The polkit module will perform 
 security tast over scale action privilegies.
```
Comment (jubalh) [inline man/lxqt-policykit-agent.1]: I have no idea what this sentence should mean. 'security tast over scale action privilegies'?

**Comment by jubalh** [comment]
I think everytime a manpage is touched one should change the header: the date and version of LXQt.

**Comment by agaida** [comment]
@jubalh - you are right :)

**Comment by palinek** [comment]
merged into @d5c2308

**Pull request closed without merging**